tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
 skipsdist=True
-envlist=py{27,36}-dj111-wag113,
+envlist=py36-dj111-wag113,
         py36-dj{111,20}-wag20,
-        py36-dj20-wag23,
-        py36-dj{20,22}-wag26,
+        py{36,37}-dj{20}-wag23,
+        py{36,37}-dj{20,22}-wag27,
```

Comment:
You mention adding support for Python 3.8 in this PR, and have added it in the Travis matrix, but haven't added it to the tox.ini file. So when tox runs, it doesn't actually test against 3.8. See for example [this latest Travis run](https://travis-ci.org/cfpb/wagtail-inventory/builds/643516761?utm_source=github_status&utm_medium=notification).

I'm not sure it's so valuable to test against 3 versions of Python -- what about testing against only 2: 3.6 (the one we're using in production) and 3.8 (the latest release)?